services/cases-api/helpers/statusCheckCondition.js

```diff
@@ -12,10 +22,46 @@ function isSignatureCompleted({ answers, people }) {
   return answers && isEncrypted(answers) && hasAllSigned(people);
```

Comment:
We are doing this check:

```js
answers && isEncyrpted(answers)
```
on multiple places. Why not adding this if-statement:

```js
  if (answers === undefined || answers === null) {
    return false;
  }
```

to the `isEncrypted` function instead?